### PR TITLE
Translated strings

### PR DESCRIPTION
**Pull request by bdorfman**
I'm planning to not merge this until we actually have translations in, but this PR has final versions of all the strings we want translated pending any feedback. 

@benzguo @jflinter if you want to give it a once over and see if there are any strings you think we should change before we send out for translation.

### CODE REVIEW

File: Stripe/STPStringUtils.h [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ *  
+ *  Completion is always called, location of range is NSNotFound with the unmodified
+ *  string if a match could not be found.
+ *
+ *  @param string     The string with tagged substrings.
+ *  @param tag        The tag to search for.
+ *  @param completion The string with the named tag removed and the range of the
+ *                    substring it covered.
+ */
++ (void)parseRangeFromString:(NSString *)string
```
Comment (jflinter) [inline Stripe/STPStringUtils.h]: Could you add a few unit tests for this?

**Comment by jflinter** [comment]
Strings look good to me!

File: Stripe/STPStringUtils.h (hunk 1 above)
Comment (bdorfman) [inline Stripe/STPStringUtils.h]: 👍

File: Stripe/STPStringUtils.h [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+//  Created by Brian Dorfman on 9/7/16.
+//  Copyright © 2016 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface STPStringUtils : NSObject
+/**
+ *  Takes a string with the named html-style tags, removes the tags,
+ *  and then calls the completion modified string and the range in it that the
```
Comment (benzguo) [inline Stripe/STPStringUtils.h]: `...calls the completion block with the...`

**Comment by benzguo** [comment]
looks good to me!

**Comment by bdorfman** [comment]
Updated from feedback and with fauxpas fixes.

I think I will probably merge this post 8.0.7 and open a new PR when translations are in since I don't want to block on that.

**Pull request merged**